### ISSUE TITLE
Projeto prático com Nuxt3

### ISSUE THREAD

**Issue by altenir** [tiagomatosweb/laravue#4]
Projeto prático com Nuxt3

- Ecossistema Vue.js

**Sobre o que é o vídeo?**
Desenvolvimento de uma aplicação SSR com Nuxt

**Exemplo prático**
Projeto prático de um CMS simples consumindo api real. Com um layout **Admin** e login e um layout de **Site** para exibir o que foi inputado pelo CMS, usando carousel, dataTable e pagination.

**Algo a mais?**
Deploy real em nuvem.

Outra coisa, nos cursos você ensina muita coisa boa e bacana, demais! (**você é o melhor**) mas seria interessante mais projeto reais com deploy etc,  não precisaria ser projetos grandes e massantes que vai tomar muito tempo nem levar meses para concluir, poderia fazer pequenos e simples com pouquissimas horas, com o principal mais usado para sites. Acredito que mais projetilhos exemplos completos vai ajudar bastante. Os cursos de introduções estão muito bons, mas acho que seria interessante fazer mais pequenos projetos (completos sem faker e menos hard-code). Me corrija se eu estiver enganado, mas é o que eu penso, aí vai valer muito apena pagar essa assinatura todo mês sabendo que vai ter onde buscar material completo pra ajudar no dia a dia, acho que muitos querem projetos de verdade, e a procura está grande por SSR, porque tem muito conteudo na internet, mas a maioria é incompleto e só introdução. 

Parabens! Você é o melhor!

**Comment by tiagomatosweb** [comment]
Valeu @altenir pela sugestão e feedback.

Só queria saber um exemplo real desse projetinho que você citou. Dá pra mandar uns 2 ou 3 exemplos com descrição cada?

**Comment by altenir** [comment]
O que eu tenho em mente no momento é esse aqui, pra entender melhor o funcionamento dos SSR com NUXT, o que é de responsabilidade do servidor? o que vai rodar no client/navegador? quando é feito uma requisição ela é executada onde?  como fazer as configurações para esse tipo de projeto? essas coisas... que o sr sabe muito bem, pra mim esse simples exemplo já ajudaria muita gente, principalmente porque você explica tudo muito bem.

**PROJETO COM NUXT3 + TAILWIND**

   **CMS (gerenciador de conteúdo de Site):**
	**SITE**:
		- Página Home com uns 2 blocos de textos e imagens que serão inseridos pela administração.
		- Form de Contato.
		- Galeria de Imagens com paginação e lightbox.
		- Galeria de Vídeo com paginação  e lightbox.
		- Área de Membro com Login.
			- Recuperação de Senha.
			- Form com dados do membro permitindo alteração (poder ser apenas uns 3 campos só, Nome, E-mail, Senha pra não ficar muito extenso).
	**ÁREA ADMINISTRATIVA**:
		- Com login usando JWT e Alteração de Senha via link.
		- Manutenção do Textos da página Home. 
		- CRUD da Galeria de imagens com filtros e busca (preload ao fazer upload da imagem).
		- CRUD da Galeria de vídeos com (links youtube por exemplo).
		- Tudo com paginação e com quantidade de registros selecionavel com um select que diz a quantidade de registros por páginas que queremos carregar.
		- Manutenção de Membros (CRUD).
	
	**DEPLOY NA VERCEL**
	
	Mais ou menos, acho que só isso já vai dá um conhecimento muito grande. Pode até tirar daí o que vai ser repetitivo.

**Comment by tiagomatosweb** [comment]
Entendi, mas isso tá longe de ser projetinho aqui e ali que não toma muito tempo haha!

Mas curti a ideia.

**Comment by altenir** [comment]
verdade, me empolguei demais kkkkk, só basico disso já daria um norte
